src/chapters/orm-relationships/exercises.rst

```diff
@@ -0,0 +1,24 @@
+Exercises: Early Bird gets the ORM!
+===================================
+
+Now that we have gotten very familiar with our ``coding-events`` application, let's design some additional features.
+As you work on your ``coding-events`` application, you may have been inspired by `Meetup <https://www.meetup.com/>`_!
+One of the cool features that Meetup has is that we can sign up for accounts to follow the events we are most interested in and keep track of our calendar of events.
```

Comment:
I think the ideas presented here would flow better if this sentence was broken down into smaller chunks.  As written, the one cool "feature" seems to be the ability to sign up for an account, rather than the ability to track interesting events.

Reply:
Done!